src/text.ts

```diff
@@ -24,19 +25,42 @@ const getTwilioClient = () => {
   return client
 }
 
-export const sendText = async ({ body, to, logger }) => {
+export const sendTwilioText = async ({ body, to, logger }) => {
   try {
     await getTwilioClient().messages.create({
       from: twilioPhoneNumber,
       to,
       body,
     })
   } catch (err) {
-    logger.fatal({ err }, "impossible to send text")
+    logger.error({ err, provider: "twilio" }, "impossible to send text")
     return
   }
 
-  logger.info({ to }, "sent text successfully")
+  logger.info({ to, provider: "twilio" }, "sent text successfully")
+}
+
+export const sendSMSalaText = async({ body, to, logger }) => {
+  try {
+    const base_url = "http://api.smsala.com/api/SendSMS"
+    const api_id = process.env.SMSALA_API_ID
+    const api_password = process.env.SMSALA_API_PASSWORD
+    const sms_type = "T"
+    const encoding = "T"
+    const sender_id = process.env.SMSALA_SENDER_ID
+    // SMSala api does not acccept nonnumeric characters like '+'
+    const phoneNumber = to.replace(/\D/g,'')
+
+    let url = `${base_url}?api_id=${api_id}&api_password=${api_password}`
+    url = url + `&sms_type=${sms_type}&encoding=${encoding}&sender_id=${sender_id}`
+    url = url + `&phonenumber=${phoneNumber}&textmessage=${body}`
+    await axios.get(url)
+  } catch (err) {
+    logger.error({ err, provider: "smsala" }, "impossible to send text")
+    return
+  }
+
+  logger.info({ to, provider: "smsala" }, "sent text successfully")
```

Comment:
`const provider = "smsala"` above the `try catch` would allow the repetition of `{ provider: "smsala" }`